src/routes/index.js

```diff
@@ -4,6 +4,7 @@ import SecureRoute from './secure-route.component';
 import LoginPage from '../pages/login/login.component';
 import Dashboard from '../pages/dashboard/dashboard.component';
 import ListUser from '../pages/list-user/list-user.component';
+import ListCompany from '../pages/list-company/list-company';
```

Comment:
I think we should use some rest api pattern here, to make the readability easier. 

For example, UsersList, CompaniesList, SomethingElseList

```suggestion
import CompaniesList from '../pages/list-company/list-company';
```